README.md

```diff
@@ -71,21 +74,26 @@ A digital datasheet shall follow the international standard notation, YYYY-MM-DD
 
 ####  4.4	 Top Level Component Specification 
 
-Source: [component.json](https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/component.json)
+Source: [component.json](https://github.com/edatasheets/digital-datasheets/blob/main/part-spec/component.json)
```

Comment:
Change path to "github.io" throughout the document

Reply:
done and fixed the script